### PR TITLE
 SK-899-update-readme-changelog-samples-delete-by-skyflow-id-in-java-sdk

### PR DESCRIPTION
**Pull request by skyflow-bharti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -833,6 +834,67 @@ Response:
...
+record.put("table", "cards");
+recordsArray.add(record);
+        
+JSONObject record2 = new JSONObject();
+record2.put("id", "2adf32e7-9a04-408e-b8bb-5b0a852422e0");
+record2.put("table", "cards");
+recordsArray.add(record2);
+
+records.put("records", recordsArray);
+DeleteOptions options = new DeleteOptions;
```
Comment (yaswanth-pula-skyflow) [inline README.md:873]: As options are optional,  need not to mention.

File: samples/src/main/java/com/example/DeleteExample.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            record.put("table", "<you_table_name>");
+            recordsArray.add(record);
+            JSONObject record2 = new JSONObject();
+
+            record2.put("id", "<your_skyflowId>");
+            record2.put("table", "<you_table_name>");
+            recordsArray.add(record2);
+
+            records.put("records", recordsArray);
+            DeleteOptions options = new DeleteOptions;
```
Comment (yaswanth-pula-skyflow) [inline samples/src/main/java/com/example/DeleteExample.java:37]: same we can ignore options

File: README.md (hunk 1 above)
Comment (skyflow-bharti) [reply README.md]: changes done

File: samples/src/main/java/com/example/DeleteExample.java (hunk 2 above)
Comment (skyflow-bharti) [reply samples/src/main/java/com/example/DeleteExample.java]: changes done

**Pull request merged**